src/lib/linux-api/src/sched.rs

```diff
@@ -50,3 +53,63 @@ pub fn sched_yield() -> Result<(), Errno> {
         .check()
         .map_err(Errno::from)
 }
+
+/// # Safety
+///
+/// Too many requirements to list here. See `clone(2)`.
+pub unsafe fn clone_raw(
+    flags: core::ffi::c_long,
+    stack: *mut core::ffi::c_void,
+    parent_tid: *mut kernel_pid_t,
+    child_tid: *mut kernel_pid_t,
+    tls: core::ffi::c_ulong,
+) -> Result<core::ffi::c_long, Errno> {
```

Comment:
nit: maybe use the same types as the syscall [implementation](https://github.com/torvalds/linux/blob/2ccdd1b13c591d306f0401d98dedc4bdcd02b421/kernel/fork.c#L3039) (`unsigned long, unsigned long, int*, int*, unsigned long`)?

Reply:
done